### PR TITLE
improve(monitor): Read ETH address from CONTRACT_ADDRESSES

### PR DESCRIPTION
**Pull request by nicholaspai**

### CODE REVIEW

File: src/monitor/Monitor.ts [hunk 1]
```diff
@@ -939,7 +956,8 @@ export class Monitor {
   private async _getDecimals(decimalrequests: { chainId: number; token: string }[]): Promise<number[]> {
     return await Promise.all(
       decimalrequests.map(async ({ chainId, token }) => {
-        if (token === ZERO_ADDRESS) {
+        const ethAddressForChain = CONTRACT_ADDRESSES[chainId]?.eth?.address ?? ZERO_ADDRESS;
```
Comment (james-a-morris) [inline src/monitor/Monitor.ts:959]: Good candidate for `getEthAddressForChain`?
```suggestion
        const ethAddressForChain = getEthAddressForChain(chainId);
```

**Review by james-a-morris — COMMENTED** [review]
Left a comment

File: src/monitor/Monitor.ts (hunk 1 above)
Comment (nicholaspai) [reply src/monitor/Monitor.ts:959]: ah shoot yeah good catch

**Review by james-a-morris — APPROVED** [review]

**Review by pxrl — APPROVED** [review]
LGTM - tested OK locally for me in the monitor.

**Pull request merged**